### ISSUE TITLE
BTS 'Manipulation' claim

### ISSUE THREAD

**Issue by grctest** [ryanRfox/Stablecoins#3]
> Manipulation – Trading volume in the Bitshares vs BitUSD market on the distributed exchange platform was low, it was therefore possible for block producers to manipulate the market by causing the value of Bitshares to fall relative to BitUSD, enabling them to obtain Bitshares at a discount.

Bitshares block producers do not have the ability to influence the price of bitshares, so this criticism is invalid, no?

**Comment by ryanRfox** [comment]
I agree. The block producers have the expectation to provide price feed data, of which the median value is used by the SmartCoin for settlement. Block producers have an incentive to publish accurate feeds, else BTS token holders will vote them out of that role.
